packages/react-components/src/components/Dialpad/Dialpad.tsx

```diff
@@ -195,7 +195,11 @@ const DialpadButton = (props: {
   const [buttonPressed, setButtonPressed] = useState(false);
 
   const dtmfToneSound = useRef<Tone>(
-    new Tone(dtmfToneAudioContext, dtmfFrequencies[digit].f1, dtmfFrequencies[digit].f2)
+    new Tone(
+      dtmfToneAudioContext,
+      dtmfFrequencies[digit as dtmfFrequenciesKeys].f1,
```

Comment:
we shouldn't cast as `as` here, instead DialpadButton props should be asking for a `dtmfFrequenciesKeys`

Reply:
This is quite important here especially since if someone passed an invalid digit into Dialpad button, this would cause a runtime exception when trying to access `.f1`